glue/viewers/common/qt/toolbar_mode.py

```diff
@@ -0,0 +1,565 @@
+"""
+ToolbarModes are CheckableTools that enable various MouseModes.
+
```

Comment:
It might be a bit too late for this, but if this is basically a modified ``mouse_mode.py`` it would have been nice to have done a ``git mv`` to preserve the history across renaming. Is there a way you can edit the history to do this?